wami/app.js

```diff
@@ -423,6 +643,29 @@ async function startApp() {
   if (!('showOpenFilePicker' in window)) {
     saveImagesButton.remove();
   }
+  
+  // Process share target data if available
+  await processShareTargetData();
+  
+  // Log protocol activation if present
+  checkProtocolActivation();
+}
+
+// Simply log any protocol activation
+function checkProtocolActivation() {
+  const url = new URL(window.location.href);
+  const protocolUrl = url.searchParams.get('url');
+  
+  // If we have a URL parameter that starts with web+wami:, it's a protocol activation
+  if (protocolUrl && protocolUrl.startsWith('web+wami:')) {
+    console.log(`Protocol activation detected: ${protocolUrl}`);
+    
+    // Just clean up the URL without any further processing
```

Comment:
```suggestion
    // Clean up the page URL.
```

I like removing words like "just" or "simply" because they tend to send the wrong signal that things are simple, which we can't assume is the case for everyone reading the code.

Reply:
Makes sense, rephrased the comments to make them more straightforward and clear.